modules/mesos-cluster/main.tf

```diff
@@ -7,6 +7,7 @@ variable "slave_instance_type" {}
 variable "subnet1" {}
 variable "subnet2" {}
 variable "subnet3" {}
+variable "mesos-cluster-cloudwatch" {}
```

Comment:
After we've set `sns_topic_arn` in mesos-cluster.tf this would change to use that variable name, so we can reference it inside the module